### PR TITLE
[docs][cloud] Memory and connection alerts

### PR DESCRIPTION
**Pull request by ddhodge**
Memory and YSQL connection alerts

@netlify /preview/yugabyte-cloud/cloud-monitor/cloud-alerts/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md [hunk 1]
```diff
@@ -127,14 +124,44 @@ If your cluster generates this alert, you may need to rate limit your queries. L
...
+High memory use could also indicate a problem and may require debugging by {{% support-cloud %}}.
+
+If memory use is continuously higher than 80%, your workload may also exceed the capacity of your cluster. Consider scaling your cluster by adding vCPUs. Refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
+
+#### Fix YSQL connection alerts
+
+YugabyteDB Managed sends a notification when the number of YSQL connections on any node in the cluster exceeds the threshold, as follows:
+
+- YSQL connections exceeds 80% of the limit (Warning).
+- YSQL connections exceeds 95% of the limit (Severe).
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md:147]: I just changed these to 60% and 95% respectively today. I have a PR out that will fix the UI text to show the right values in the alert configuration.

**Comment by daniel-yb** [comment]
@vitroth can you please take a look here as well?

**Comment by daniel-yb** [comment]
On memory:
- Users can also consider scaling their cluster horizontally, not just vertically. More nodes will lower the workload per-node on average if the issue isn't a single query that consumes a lot of memory slamming a single tablet

On connection limits:
- Users can also consider scaling their cluster horizontally, not just vertically. Scaling horizontally will also give them more total connections since the overall cluster limit is defined as "per node limit * number of nodes"
- Would be good to call out somewhere in here that connection limits are actually associated to vCPU count when you recommend they can look to scale up if they are seeing issues.

**Comment by ddhodge** [comment]
@vitroth can you please review

**Review by vitroth — APPROVED** [review]

**Pull request merged**